src/cephadm/cephadmlib/daemon_identity.py

```diff
@@ -33,6 +33,10 @@ def daemon_type(self) -> str:
     def daemon_id(self) -> str:
         return self._daemon_id
 
+    @property
+    def ceph_name(self) -> str:
+        return f'{self.daemon_type}.{self.daemon_id}'
```

Comment:
NIT: The `<type>.<id>`  format is already used a lot as "daemon name" so I'd prefer that over "ceph name" unless we have some sort of conflict with something else called "daemon name" here. For example, a bunch of the type specific classes like the `CephIscsi` class have a `get_daemon_name` function that builds a name with this exact same format. It's also not limited to ceph daemons. The `Haproxy` and `Keepalived` class use the same convention.

Reply:
Will do

Reply:
done.